tests/smoke-tests/src/main/resources/servers/toUpgradeTest/etc/artemis.profile

```diff
@@ -0,0 +1,63 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+ARTEMIS_HOME='/Users/someUser/Downloads/upgrade/apache-artemis-2.25.0'
```

Comment:
I just thought it would be nice to go a few versions before.

Reply:
Probably worth the docs clarifying what is allowed for this, since differnet versions might have made different changes (and did in this case...with the upgrade not making them all, at least in the version that existed when I commented...might chagne given your latest comments)

Reply:
I was thinking similar, my understanding is this tool should work back to the last special upgrade procedure in 2.21

Reply:
@TheCycoONE , @gemmellr I don't know how to write something that would read simple though.

I prefer to just say the upgrade tool is used to upgrade a server.

If you think some good wording to add here... can you send a separate PR in there?

Reply:
Fair, yes.